ELISA Policies and Guidelines

```diff
@@ -0,0 +1,69 @@
+# ELISA Policies and Guidelines
+
+This document outlines the policies and guidelines for developing and publishing ELISA deliverables.
+
+**ELISA deliverables:**
+- ELISA develops an example qualitative analysis for safety critical use-cases for the Linux kernel e.g: Automotive and Medical.
+- ELISA provides resources for System integrators to apply and use to analyze qualitatively and quantitatively on their systems. For example, ELISA will analyze published CWEs to identify hazards for the two use-cases.
```

Comment:
@shuahkh I believe this was partially taken from the technical charter and could also be found in a previous version of the gold deck. I propose to do rewording on the ELISA deliverables. 

What about creating an issue to update this part of the documentation after we have merged this. Alternative could also be to put another commit on top of this PR to correct it.